domain/src/main/java/com/example/domain/Domain.kt

```diff
@@ -0,0 +1,4 @@
+package com.example.domain
+
+class Domain {
+}
```

Comment:
add an empty line at the end of the file pls

Reply:
done!